src/main/java/com/example/Huiswerkles10backend/services/TelevisionService.java

```diff
@@ -0,0 +1,194 @@
+package com.example.Huiswerkles10backend.services;
+
+import com.example.Huiswerkles10backend.Exceptions.RecordNotFoundException;
+import com.example.Huiswerkles10backend.dtos.input.TelevisionInputDto;
+import com.example.Huiswerkles10backend.dtos.output.TelevisionDto;
+import com.example.Huiswerkles10backend.model.Television;
+import com.example.Huiswerkles10backend.repository.RemoteControllerRepository;
+import com.example.Huiswerkles10backend.repository.TelevisionRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class TelevisionService {
+
+    //constructor injection
+    private final TelevisionRepository repos;
+    private final RemoteControllerRepository rcrepos;
+
+    public TelevisionService(TelevisionRepository repos, RemoteControllerRepository rcrepos) {
+        this.repos = repos;
+        this.rcrepos = rcrepos;
+    }
+
+
+    //aanpassen naar getTelevison methode. CRUD repository geeft een iterable
+    public List<TelevisionDto> getAllTelevisions() {
+        Iterable<Television> televisions = repos.findAll();
+        List<TelevisionDto> televisionsOutputDtos = new ArrayList<>();
+        for (Television t : televisions) {
+            TelevisionDto tdto = new TelevisionDto();
```

Comment:
Waarom maak je hier (en op meerdere plekken) een nieuwe dto aan die je veld voro veld vult, terwijl je daar een helper methode voor hebt gemaakt "transferEntityToDto"? Dat is onnodig veel herhaling en schrijfwerk